sale_financial_risk/tests/test_payment_financial_risk.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2023 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.fields import Command
+from odoo.tests import tagged
+from odoo.tools import mute_logger
+
+from odoo.addons.payment.tests.common import PaymentCommon
+from odoo.addons.payment.tests.http_common import PaymentHttpCommon
+
+
+@tagged("-at_install", "post_install")
+class TestRiskSalePayment(PaymentCommon, PaymentHttpCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(
+            context=dict(cls.env.context, tracking_disable=True, mail_create_nolog=True)
+        )
```

Comment:
I have this more complete one for your reference:

```suggestion
        cls.env = cls.env(
            context=dict(
                cls.env.context,
                mail_create_nolog=True,
                mail_create_nosubscribe=True,
                mail_notrack=True,
                no_reset_password=True,
                tracking_disable=True,
            )
        )
```

Reply:
Thanks :) Included